ucsb-cs156-f24/team02-f24-06
# Added UCSBOrganization create page and tests

- Adds Create page for UCSBOrganization
- Adds Tests for Create page
- Passes all tests

To Test:
The page should look like this at first:
<img width="1371" alt="Screenshot 2024-11-06 at 11 03 04 AM" src="https://github.com/user-attachments/assets/76df356d-5e3c-4459-ac48-01a888278fcb">

Don't input anything, just press create, it should look like this:
<img width="1338" alt="Screenshot 2024-11-06 at 11 03 10 AM" src="https://github.com/user-attachments/assets/df5ab1fd-52bc-448f-9b7c-94387b941098">

After you fill out all fields and press create, there should be a pop up on the top right similar to the one below:
<img width="1569" alt="Screenshot 2024-11-06 at 11 24 07 AM" src="https://github.com/user-attachments/assets/1ff4242a-35fe-40c3-8034-62de67b9bdff">

Closes #22
